include/aos/sm/cni.hpp

```diff
@@ -277,6 +277,15 @@ class CNIItf {
      * @return Error.
      */
     virtual Error ValidateNetworkList(const NetworkConfigList& net) = 0;
+
+    /**
+     * Returns network list from cache.
+     *
+     * @param[out] net Network list.
+     * @param[out] rt Runtime configuration.
+     * @return Error.
+     */
+    virtual Error GetNetworkListCachedConfig(NetworkConfigList& net, RuntimeConf& rt) = 0;
```

Comment:
Shouldn't net parameter be input and const?

Reply:
No, they shouldn’t be like that because they are filled inside this method.